api/candlepin-api-spec.yaml

```diff
@@ -5564,6 +7371,47 @@ components:
             upstreamConsumer:
               $ref: '#/components/schemas/UpstreamConsumerDTO'
 
+    OwnerInfo:
+      description: Represents an owner information
```

Comment:
Minor grammatical mistake here. Needs either a noun or to drop the "an":

"Represents owner information", or "Represents an owner-information container"

Reply:
Done.